### ISSUE TITLE
APA formatted bibliography

### ISSUE THREAD

**Issue by mvuorre** [mvuorre/quarto-apaish#9]
With long documents Typst seems to break with an error related to `#emph`.

- [x] Add a reproducible example / test
- [x] Figure out what is broken and send appropriate issue
- [x] Format the bibliography to APA format (at least a hanging indent or something)

**Comment by mvuorre** [comment]
There's actually a simple fix ([f828696](https://github.com/mvuorre/quarto-apaish/commit/f828696b042371856e9ca1ab56c29ac87c24cde7)):

<pre>
---
title: APA7ish Quarto Typst PDF template
runninghead: APA7ish

author:
  - name: Norah Jones

format:
  quarto-apaish-typst: default
---

[@cohenProblemUnitsCircumstance1999]

```{=typst} 
#bibliography("test.bib", title: "References", style: "apa")
```
</pre>

The downside of this is that it doesn't generalize to other Quarto formats (.docx, .html, etc), and so a general (YAML-ish) option should be preferred. Keeping this open but will edit related bits on main.

**Comment by mvuorre** [comment]
Typst will support csl files natively in the future, closing.
